Scenes/ContentManager/Custom_Editors/TacticalMapEditor/Scripts/TacticalMapEditorTileBrush.gd

```diff
@@ -23,3 +33,28 @@ func set_selected(is_selected: bool) -> void:
 		modulate = Color(0.227, 0.635, 0.757)
 	else:
 		modulate = Color(1,1,1)
+
+
+# Set the label text and adjust the size of the label and the tile sprite accordingly
+func set_label(text: String):
+	label.text = text
+	label.custom_minimum_size = Vector2(64, 12)
+
+	label.autowrap_mode = TextServer.AUTOWRAP_WORD_SMART  # Enable word wrapping
+	label.size_flags_horizontal = Control.SIZE_FILL
+	label.size_flags_vertical = Control.SIZE_SHRINK_END
+	label.custom_minimum_size = Vector2(64, label.get_minimum_size().y)
+
+	# Update the minimum size of the parent container
+	update_min_size()
+
+
+func _on_label_minimum_size_changed():
+	# Update the minimum size of the parent container
+	update_min_size()
+
+
+# Update the minimum size of the parent container
+func update_min_size():
+	var total_height = 64 + label.custom_minimum_size.y
```

Comment:
Again 64 + another magic number 20 (below). It would be nice to label that so person reading know whats going on.

Right now I have no idea what this total_height+20 is really doing :(

Reply:
Added a comment to explain the 20 height